eu.stethewwolf.gresistor.json

```diff
@@ -0,0 +1,31 @@
+{
+    "app-id": "eu.stethewwolf.gresistor",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "gresistor3",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "modules": [
+        {
+            "name": "gresistor",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=${FLATPAK_DEST}",
+                "install -Dm644 eu.stethewwolf.gresistor.desktop -t ${FLATPAK_DEST}/share/applications",
+                "install -Dm644 gresistor3/pixmaps/48x48/eu.stethewwolf.gresistor.png  -t ${FLATPAK_DEST}/share/icons/hicolor/48x48/apps/",
+                "install -Dm644 gresistor3/pixmaps/128x128/eu.stethewwolf.gresistor.png  -t ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/"
```

Comment:
you never install the appstream file.

Reply:
I was hoping that the line 31 in the [setup.py](https://github.com/stethewwolf/gResistor/blob/main/setup.py) was enoughth

Reply:
no sure but what I can say is that setup.py install the icons in the wrong place. and you shouldn't have to install the .desktop here since it does it too.